README.md

```diff
@@ -17,12 +17,8 @@ Role Variables
 All variables are optional, see `defaults/main.yml` for the full list
 
 OMERO.server version.
-- `omero_server_release`: The OMERO release, e.g. `5.2.2`.
-This defaults to `latest`, but due to the broken registry a proper upgrade check has not been implemented so this will not have the expected effect (it will always attempt to upgrade even if the current server is the latest).
-You are advised to change this to an actual release version.
-- `omero_server_upgrade`: Upgrade OMERO.server if the current version does not match `omero_server_release`.
-  This is a workaround for the inability to check for the latest version when `omero_server_release: latest`.
-  At present this will skip an available upgrade if `False` but in future this will be changed to fail if the currently installed version does not match `omero_server_release`.
+- `omero_server_release`: The OMERO release, e.g. `5.2.2`, default `latest`.
+  You are advised to change this to a release version for production systems, otherwise your server will be automatically upgraded when a new release is made.
```

Comment:
This slightly worries me. I'd almost prefer to put nothing and then print an error:

```
omero_server_release must be set to either "latest" or a concrete version.
```

Reply:
I'd second that. No default of latest would be preferable to me, too.

Reply:
`present`: very nice!